src/daemon/daemon.cpp

```diff
@@ -3671,6 +3673,10 @@ void mp::Daemon::run_commands_at_boot_on_instance(const std::string& name, fmt::
 
             warned_exec_failure = true;
         }
+        catch (const ExitlessSSHProcessException&) // Timeout does not mean there is a configuration error.
```

Comment:
Why not just use the string returned in `what()` here and not have to capture the current command above?